app/views/docs/functions-runtimes.phtml

```diff
@@ -0,0 +1,73 @@
+<?php
+
+use Appwrite\Utopia\View;
+
+$events = $this->getParam('events', []);
+$runtimes = $this->getParam('runtimes', []);
+
+$sorted_runtimes = [];
+
+foreach ($runtimes as $key => $item) {
+    $name = $item['name'];
+    
+    if (!isset($sorted_runtimes[$name])) {
+        $sorted_runtimes[$name] = [];
+    }
+
+    $item['version'] = $key;
+
+    $sorted_runtimes[$name]['versions'][] = $item;
+}
+
+$sorted_runtimes['Node.js']["cloud"] = true;
+$sorted_runtimes['PHP']["cloud"] = true;
+$sorted_runtimes['Ruby']["cloud"] = true;
+$sorted_runtimes['Python']["cloud"] = true;
+$sorted_runtimes['Dart']["cloud"] = true;
+?>
+
+
+<p>
+    Appwrite Functions supports an extensive list of runtimes to meet your unique tech preferences.
+    Not all runtimes are available on Appwrite Cloud, check for the <span class="tag green">Cloud</span> label in each listed runtime to know which ones are available.
+</p>
+
+<h2><a href="/docs/functions#supportedRuntimes" id="supportedRuntimes">Supported Runtimes</a></h2>
+
+<p>Below is a list of supported Functions runtimes. The Appwrite team continually adds support for new runtimes.</p>
+
+<table cellspacing="0" cellpadding="0" border="0" class="full margin-bottom-large">
+    <thead>
+        <tr>
+            <th style="width: 50px"></th>
+            <th style="width: 80px">Name</th>
+            <th>Image</th>
+            <th  style="width: 100px">Architectures</th>
+            <th  style="width: 140px">Platforms</th>
+        </tr>
+    </thead>
+    <tbody>
+        <?php foreach($sorted_runtimes as $key => $runtime): ?>
+        <tr>
+            <td>
+                <img src="" data-ls-attrs="src=/images/runtimes/<?php echo $this->escape($runtime['versions'][0]['logo'] ?? ''); ?>" alt="Function Env." class="avatar xxs" />
+            </td>
+            <td>
+                <?php echo $this->escape($key); ?>
+            </td>
+            <td>
+            <?php foreach($runtime['versions'] as $key => $version): ?>
```

Comment:
Here used to be a inline-code of runtime name that you can copy and paste into your self-hosted appwrite, to support it. I think we really want to keep that. Its much more valuable than link to Docker Hub of image

Reply:
![CleanShot 2023-08-27 at 13 51 41@2x](https://github.com/appwrite/docs/assets/19310830/7ece66f6-06ad-4640-b2b2-59631566bb33)